src/js/CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
 if(LINUX)
     # to find libopenvino_genai.so in the same folder
     set(rpaths "$ORIGIN")
+    list(APPEND rpaths "$ORIGIN/../../openvino-node/bin")
```

Comment:
Does that change how the project should be built described in https://github.com/openvinotoolkit/openvino.genai/blob/3ea96c05d89c2c6aaa354df48fabe48eb0168166/src/js/README.md? As far as I understand, it's no longer required to use OPENVINO_EXTRA_MODULES. If that's true, you can probably merge the instructions with https://github.com/openvinotoolkit/openvino.genai/blob/3ea96c05d89c2c6aaa354df48fabe48eb0168166/src/docs/BUILD.md

It's true that there' a disclaimer
> Tested on Ubuntu, another OS didn't tested yet

But I was hoping that it'll go away after some time. The problem is that rpath is Linux specific